package.json

```diff
@@ -30,7 +30,7 @@
     "localforage": "1.7.3",
     "localforage-observable": "2.0.0",
     "mark.js": "8.11.1",
-    "marked": "github:mattermost/marked#3c10e2109527cd14a76e1f8edd9d317d8b8988b0",
+    "marked": "github:mattermost/marked#3a9d0a5bd6d735df1a716ab50f73d58a1d865bdb",
```

Comment:
I don't see a test in marked's commit.  Please add there or here in `/link_only_renderer.test.jsx`, something like:
```javascript
{
    description: 'link: link with curly brackets',
    inputText: 'Let\'s try http://example/result?things={stuff}',
    outputText: 'Let&#39;s try <a class="theme markdown__link" href="http://example/result?things={stuff}" target="_blank">http://example/result?things={stuff}</a>',
},
```

Reply:
Thanks for your suggestion. Test case has been added!